src/components/WrapEtherBtn.tsx

```diff
@@ -270,6 +270,17 @@ const WrapUnwrapEtherBtn: React.FC<WrapUnwrapEtherBtnProps> = (props: WrapUnwrap
                     required: 'The amount is required',
                     min: 0,
                   })}
+                  onBlur={(e): void => {
+                    // react-hook-form does something onBlur that interferes with button clicks
+                    // at the same time modali buttons rerender at every opportunity
+                    // so click never arrives where we expect it to
+                    // at least that is my guess
+                    const { relatedTarget } = e
+                    // here be hacks
+                    if (relatedTarget instanceof HTMLButtonElement) {
+                      relatedTarget.click()
```

Comment:
oh boy! 🙈

it must be the validation that is done onBlur, but if the validation is successful, it shouldn't intercept the click.
Probably it'll be nice to do a codepen for them to show this weird thing, they must have an official explanation.

Thanks for debuging this one. It was not an easy one!